app/views/org_admin/phases/container.html.erb

```diff
@@ -8,7 +9,7 @@
 <div class="row">
   <div class="col-md-12">
     <!-- render navigation tabs for the template-->
-    <%= render partial: "/org_admin/templates/navigation", locals: local_assigns %>
+    <%= render partial: "/org_admin/templates/navigation", locals: local_assigns.merge({ modifiable: modifiable }) %>
```

Comment:
since we do this .merge each time we render a partial, why not just save the change to local_assigns?